### DISCUSSION TITLE
i'm having problems downloading my videos

### DISCUSSION THREAD

**Discussion by GuiSilva012** [Tyrrrz/YoutubeDownloader#277] [Q&A]
![Sem título](https://user-images.githubusercontent.com/90194157/178974509-c3b83a89-220e-4347-b698-b6fe1174809e.png)
I've been trying since yesterday to download my videos but this message is always appearing, did I do something wrong or is it a general problem?

**Comment by Tyrrrz** [comment]
Does it happen on all videos?

**Reply by GuiSilva012** [reply]
yes, in all

**Reply by GuiSilva012** [reply]
started working again! ^^
